client/src/controllers/index.ts

```diff
@@ -1,10 +1,12 @@
 import type { Definition } from '@hotwired/stimulus';
 
 import { AutoFieldController } from './AutoFieldController';
+import { ActionController } from './ActionController';
 
 /**
  * Important: Only add default core controllers that should load with the base admin JS bundle.
  */
 export const coreControllerDefinitions: Definition[] = [
   { controllerConstructor: AutoFieldController, identifier: 'w-auto-field' },
+  { controllerConstructor: ActionController, identifier: 'w-action' },
```

Comment:
It would be good to add these in alphabetical order by constrictor name - will make this file easier to navigate when there are lots here.